applications/portal/frontend/src/components/DeleteExperimentDialog.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from "react";
+import { Button, Dialog, DialogActions, DialogContent, DialogTitle, IconButton } from "@material-ui/core";
+import CLOSE from "../assets/images/icons/close.svg";
+import { Modal as IModal } from "../types/modal";
+import { Box, makeStyles, Typography } from "@material-ui/core";
+import { headerBorderColor, headerButtonBorderColor, secondaryColor, switchActiveColor } from "../theme";
+import workspaceService from "../service/WorkspaceService";
+import DeleteDialog from "./common/ExperimentDialogs/DeleteModal";
+
+
+const useStyles = makeStyles(() => ({
+  delete: {
+    '& .details': {
+      flexGrow: 1,
+      '& .detail-block + .detail-block': {
+        marginTop: '1.5rem',
+      },
+
+      '& .MuiTypography-root': {
+        fontSize: '0.75rem',
+        lineHeight: '0.9375rem',
+        fontWeight: 600,
+        color: secondaryColor,
+        marginBottom: '0.5rem',
+      },
+
+      '& p': {
+        '&.MuiTypography-root': {
+          color: headerButtonBorderColor,
+          fontWeight: 400,
+        },
+      },
+
+    },
+  }
+}));
+
+
+
+export const DeleteExperimentDialog = (props: any) => {
+  const { open, handleClose, experimentId, refreshExperimentList } = props;
+  const classes = useStyles();
+  const api = workspaceService.getApi();
+
+  const deleteExperiment = async (id: string) => {
+    if (id) {
+      const response = await api.destroyExperiment(id)
+      if (response) {
```

Comment:
issue: @D-GopalKrishna shouldn't we check the response upon yes/no success? In case no success then imho we should show the error/issue

Reply:
@zsinnema thanks for pointing this out. I also added a backend authorization check on top - to check if the requested user and the owner of the experiment are one and the same.